cloudigrade/account/tests/test_tasks.py

```diff
@@ -431,11 +452,18 @@ def test_run_inspection_cluster_with_no_instances(self, mock_boto3):
         mock_boto3.client.return_value = mock_ecs
 
         with self.assertRaises(AwsECSInstanceNotReady):
-            tasks.run_inspection_cluster([Mock()])
+            tasks.run_inspection_cluster(messages)
 
+    @patch('account.models.MachineImage.objects')
     @patch('account.tasks.boto3')
-    def test_run_inspection_cluster_with_too_many_instances(self, mock_boto3):
+    def test_run_inspection_cluster_with_too_many_instances(
+            self, mock_boto3, mock_machine_image_objects):
+        """Assert that an exception is raised with too many instances."""
```

Comment:
It looks like this might be a duplicated docstring (there is an identical one on line 464 in this same function)

Reply:
Good eye, fixed~